ui/src/app/pages/admin/admin-user-profile.tsx

```diff
@@ -398,8 +406,10 @@ const DisabledToggle = (props: {
   currentlyDisabled: boolean;
   previouslyDisabled: boolean;
   toggleDisabled: () => void;
+  disabled?: boolean;
```

Comment:
I don't like this because we're already using the word "disabled" to mean something else.  We can use `isCurrentUser()` directly on line 426